src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/ManagingOfficerIndividualValidator.java

```diff
@@ -96,6 +100,12 @@ private boolean validateNationality(String nationality, Errors errors, String lo
                 && StringValidators.isValidCharacters(nationality, qualifiedFieldName, errors, loggingContext);
     }
 
+    private boolean validateSecondNationality(String secondNationality, Errors errors, String loggingContext) {
```

Comment:
would you be able to add tests to check the valid chars part of the second nationality validation? in the MO individual and BO individual

Reply:
Probably it is not part of this story, up to you @mattch1 (not a blocker), but we need to make sure that the user does not select the same nationality as the first one

Reply:
There are tests for that. Yes we should should check that the nationalities are different. Good point.

Reply:
it was sonar reporting that a condition was missed, thought it might have been the invalid chars. It could be that there is no test for a valid second nationality, try including it in the 'no errors reported' test

Reply:
More testing with the check equal now up to 100%